### PR TITLE
[Ground.0.01] first release of coq-ground

### PR DESCRIPTION
**Pull request by JoanThibault**
Ground (Generic and Reusable Organized Usefull Natural Definitions) A usefull extension to Coq's standard library

### CODE REVIEW

File: released/packages/coq-ground/coq-ground.0.01.0/opam [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+authors: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+license: "LGPL-3.0-only with OCaml-LGPL-linking-exception"
+homepage: "https://gitlab.com/boreal-ldd/ground"
+bug-reports: "https://gitlab.com/boreal-ldd/ground/-/issues"
+depends: [
+  "coq" {>= "8.14.0"}
+]
+build: [make]
+install: [make "install"]
+remove: [make "uninstall"]
```
Comment (palmskog) [inline released/packages/coq-ground/coq-ground.0.01.0/opam:14]:
```suggestion
build: [make "-j%{jobs}%"]
install: [make "install"]
```
I really recommend allowing parallel build. And opam doesn't need an uninstall, that is already taken care of.

File: released/packages/coq-ground/coq-ground.0.01.0/opam [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+dev-repo: "git+https://gitlab.com/boreal-ldd/ground.git"
+url {
+	src:"hhttps://gitlab.com/boreal-ldd/ground/-/archive/v0.01/ground-v0.01.tar.gz"
+	checksum:"md5=eae5b7c0df663a2d34c3b67ffccb9c7d"
```
Comment (palmskog) [inline released/packages/coq-ground/coq-ground.0.01.0/opam:18]:
```suggestion
	src:"https://gitlab.com/boreal-ldd/ground/-/archive/v0.01/ground-v0.01.tar.gz"
	checksum:"sha512=d2814b5eac7b7ac7619fd86007742ee98bc94f57f3dc34c3b4d5e6ad0c0ec4aecdfff5e243921a464bb7498b1c9ed1458a337b9006b82e70ab92d8d6c24a8748"
```
Fix URL and replace MD5.

**Review by palmskog — CHANGES_REQUESTED** [review]

**Review by palmskog — APPROVED** [review]

**Comment by JoanThibault** [comment]
thank you very much =)

File: released/packages/coq-ground/coq-ground.0.01.0/opam [hunk 3]
```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+synopsis: "Ground : A Usefull Extension to Coq's Standard Library"
```
Comment (gares) [inline released/packages/coq-ground/coq-ground.0.01.0/opam:2]: I think it is spelled "useful", with just one "l"

**Pull request merged**